Mathlib/Algebra/Order/Positive/Ring.lean

```diff
@@ -40,22 +40,22 @@ theorem coe_add (x y : { x : M // 0 < x }) : ↑(x + y) = (x + y : M) :=
 
 instance addSemigroup : AddSemigroup { x : M // 0 < x } :=
   Subtype.coe_injective.addSemigroup _ coe_add
-#align subtype.add_semigroup Positive.addSemigroup
+#align positive.subtype.add_semigroup Positive.addSemigroup
```

Comment:
Is this right?

Reply:
https://leanprover-community.github.io/mathlib_docs/algebra/order/positive/ring.html#positive.subtype.add_semigroup